redfish-core/lib/systems.hpp

```diff
@@ -989,6 +992,13 @@ inline void
                 return;
             }
 
+            // Return nothing when power restore policy set to "None"
+            if (policyMapsIt->second == "None")
+            {
+                aResp->res.jsonValue["PowerRestorePolicy"] = "";
```

Comment:
I am pretty sure this will throw a Redfish Validator error

Reply:
Isnt the validator run during CI? I didnt see a fail on the upstream change

Reply:
ok, thanks...

Upstream discussion is leaning towards possibly removing the `PowerRestorePolicy` when its "None", so will update and re-test that scenario.

Reply:
Updated according to consensus upstream to return "AlwaysOff" for a `PowerRestorePolicy` = "None"